docs/modules/ROOT/pages/references/security.adoc

```diff
@@ -0,0 +1,120 @@
+= Security
+
+== Access Management
+
+=== Long Lived Tokens
+
+The default OpenShift 10 year emergency admin token is disabled and replaced by a short lived token.
+
+See xref:oc4:ROOT:references/architecture/emergency_credentials.adoc[].
+
+=== Two Factor Authentication
+
+VSHN managed OpenShift clusters are configured to use two factor authentication through the VSHN SSO for the web console by default.
+The configuration is centrally managed according to best practices.
+
+See xref:oc4:ROOT:references/architecture/single_sign_on.html[].
+
+=== Sudo
+
+VSHN managed OpenShift clusters are configured to deny dangerous operations by default.
+
+See xref:oc4:ROOT:explanations/sudo.adoc[].
+
+== Containers
+
+=== Priviledged Containers and Build Strategies
+
+VSHN managed OpenShift clusters are configured to deny the use of privileged containers and build strategies by default.
+
+See xref:oc4:ROOT:explanations/pod_security.adoc[].
+
+=== Seccomp
+
+VSHN managed OpenShift clusters are configured to use the CRI-O seccomp profile `runtime/default` by default.
+
+This restricts some syscalls that could shorten exploit chains.
+
+== Network
+
+=== Load Balancer
+
+VSHN managed OpenShift clusters include, depending on the cloud provider, hardened load balancers based on HAProxy.
+
+Addictional firewalls and jumphosts can be configured on request.
+
+=== Cilium
+
+VSHN managed OpenShift clusters uses the hardened enterprise version of Cilium as the default network plugin.
```

Comment:
Should we also link to https://products.vshn.ch/openshift/cilium.html or is that more sales documentation?